### DISCUSSION TITLE
Show valve opening in percent

### DISCUSSION THREAD

**Discussion by itn3rd77** [SukramJ/aiohomematic#87] [Ideas]
Hi Markus,

what do you think about the proposal to show the valve opening for HmIP-eTRV* and HM-CC-RT-DN in precent and not on a fraction of 0 to 1? It's a bit easier to read for human eyes. Would that be possible?

Best regards
Ingo

**Comment by SukramJ** [comment]
Basically you are  right and i am aware of this. 
This is an auto generated entity in hahomematic, where no dev has to do anything to get this working. 
I personally won't create an implementation specific for "LEVEL", but i have something more generic on my list.

**Reply by SukramJ** [reply]
Now i remember why it's not implemented: This is a [number entity](https://developers.home-assistant.io/docs/core/entity/number). A number entity is **editable**, but it has no unit.
Creating a sensor, which is **not editable** but has a **unit** requires additional effort.

**Reply by SukramJ** [reply]
Latest version support display valve state in percentage.

**Reply by itn3rd77** [reply]
Great thanks a lot!
